dapps/marketplace/src/components/MobileModalHeader.js

```diff
@@ -34,21 +35,31 @@ function MobileModalHeader({
     headerClassList = headerClassList.concat(className.split(' '))
   }
 
+  let closeButton
+
+  if (showCloseButton) {
```

Comment:
nitpick: 
```
if (showCloseButton || showBackButton) {
closeButton = (
      <a className={`modal-action-button ${showCloseButton ? 'close-button' : 'back-button'}`} onClick={onBack} />
    )
}
```

Reply:
Thanks. We can probably improve this thing further. Instead of having two boolean parameters (that are mutually exclusive), I wonder if we can have a single parameter that takes a one of string (constants/enums) values and render based on that.